mock/py/mockbuild/util.py

```diff
@@ -582,7 +582,7 @@ def logOutput(fdout, fderr, logger, returnOutput=1, start=0, timeout=0, printOut
                         if line != '':
                             line = ansi_escape.sub('', line)
                             if fderr is s and not line.startswith('+ '):
-                                logger.debug("BUILDSTDERR: %s", line)
+                                logger.debug("{0} {1}".format(logging.getLogger('mockbuild').stderr_line_beginning, line))
```

Comment:
As you mentioned in face to face discussion, there is going to be a space at the beginning of the line if no `stderr_line_beginning` is specified. What about having `{0}{1}` without space and letting users have a space on the end of their prefix.

Reply:
Please do not use format() as it is called always, even when log is not actually used. See https://stackoverflow.com/questions/4148790/lazy-logger-message-string-evaluation